input/pagecontent/reference.md

```diff
@@ -967,127 +1164,51 @@ Each rule in the rule set should be compatible with item where the rule set is i
   * insert FranceObservationRuleSet
   ```
 
+#### Obeys Rules
 
-#### Slicing Rules
+[Invariants](https://www.hl7.org/fhir/conformance-rules.html#constraints) are constraints that apply to one or more values in instances, expressed as [FHIRPath expressions](https://www.hl7.org/fhir/fhirpath.html). An invariant can apply to an instance as a whole or a single element. Multiple invariants can be applied to an instance as a whole or to a single element. The syntax for applying invariants in profiles is:
```

Comment:
Well they don't have to be FHIRPath right?

Reply:
Added xpath.